extensions/amp-soundcloud/1.0/test/validator-amp-soundcloud.html

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
Note: This file is not needed until we allow `1.0` in the validation rules for this component (`extensions/amp-soundcloud/validator-amp-soundcloud.protoascii`). To do this I would recommend removing this file and:
- modifying the spec in the above file, using `satisfies` and `excludes` to differentiate restrictiveness between `0.1` and `1.0` versions
- copying the existing `validator-amp-soundcloud.html` and `validator-amp-soundcloud.out` files from the `0.1/test` directory to the `1.0/test` directory
- manually updating the script imported from the copied files to `1.0` 

Up to you whether you would like to do all that in this same PR or in a separate one. #33478 is a good reference PR you can follow which does exactly what I described here.

Reply:
Thank you so much for the feedback. Let me go through and update all of them. I would like to make changes to the same PR, so I can make notes for future Bento components.